airbyte-cdk/python/airbyte_cdk/sources/declarative/parsers/model_to_component_factory.py

```diff
@@ -682,6 +684,12 @@ def _merge_stream_slicers(self, model: DeclarativeStreamModel, config: Config) -
             and model.retriever.partition_router
         ):
             stream_slicer_model = model.retriever.partition_router
+
+            # if model.incremental_sync and hasattr(model.incremental_sync, "global_parent_cursor") and model.incremental_sync.global_parent_cursor:
```

Comment:
is this commented code block still required?

Reply:
Fixed